### ISSUE TITLE
Updated App design

### ISSUE THREAD

**Issue by bumi** [getAlby/alby-installer-macos#2]
We got this design suggestions for the app
![image](https://user-images.githubusercontent.com/318/148686864-c78ca85b-a47c-4ef5-b7b2-f187711edc8f.png)
![image (1)](https://user-images.githubusercontent.com/318/148686859-67094392-7851-40ad-b5a6-fbb5535a64f6.png)
.
We must always reuse any default macOS style and not do anything custom but I guess we can add some background image/color the logo etc. 

Please also note that the amount of browsers shown depends on the browser installed on the user's system.

**Comment by stuffmc** [comment]
These are great designs, but:

- How does it look in Dark mode?
- These aren't the Chrome/Chromium Icons on macOS*

No more "1" and "2" like on your original "design"?

* We certainly can package the Icons in the App, but I'd rather keep on reading it from the App as those icons might change in the future and confuse the users. One option would be to pick the same background as the Chrome Icon. Not sure if that would work and the size of the Icons would also be different _anyways_.

**Comment by bumi** [comment]
These images are just inspiration. We do not want to package the icons and keep it as you have built it. that is best!

So here is mainly the idea to add that header with the Alby logo and a bit text I guess. 
Leaving this up to the designers :)

**Comment by stuffmc** [comment]
I don't think we need more designers than the designs that are there. We have all the necessary. I prefer design 1, since the straight buttons are slightly boring. Let's make that happen as part of the next release.

**Comment by bumi** [comment]
so design 1 it is. 
* we use the Browser logos as you already load it
* we need to use the proper Alby logo 
* make sure it works on dark mode

**Comment by stuffmc** [comment]
Reopen this because right now it doesn't work in Dark mode because of `.preferredColorScheme(.light)` (which should be removed in the future — this is a downgrade) and the `Color(` in the code. We shall not use `Color` in the code but instead Color Assets that will adapt automatically.

**Comment by bumi** [comment]
@johnsBeharry should we use there this logo or the one we have everywhere with the normal eyes?

**Comment by stuffmc** [comment]
Oh good catch, @bumi! I didn't see that. I prefer the one with the real eyes, and it's also the one on https://getalby.com. As for the dark mode, I've added a first version of it, which we could change (especially the background color), and I think the Icon should be available with a white background instead of black, potentially. Speaking of Darkmode, I don't think the HTML/CSS in the extension respects Dark mode, but that'd be another ticket.

**Comment by johnsBeharry** [comment]
@bumi i think the idea behind it was that it's for connections / messaging. for consistency though we could. Let me speak with Alex about the branding guidelines
